.github/workflows/build-artifacts-code.yml

```diff
@@ -21,6 +21,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
MinVer need tags to work. It's a known issue with `actions/checkout` - this action has no explicit option to fetch tags. `fetch-depth: 0` effectivelty fetches all tags. Yep, it fetches all commits as well 😞 .

Reply:
fixed in https://github.com/actions/checkout/pull/579